Add information on extension footprint analyzer (#8962)
magento/devdocs: src/marketplace/sellers/extension-footprint.md

---
group: marketplace-sellers

> title: Extension Footprint analysys

Reviewer: ```suggestion
title: Extension Footprint Analysis
```

---

> {: .bs-callout .bs-callout-info}Footprint analyzer is currently released as a beta feature meaning that some of the information may not be copletely accurate or present at all. Beta status of the feature is also indicated on the product lisging page.

Reviewer: Put text on next line. It should not be on the same line as content in braces.

> {: .bs-callout .bs-callout-info}Footprint analyzer is currently released as a beta feature meaning that some of the information may not be copletely accurate or present at all. Beta status of the feature is also indicated on the product lisging page.

Reviewer: ```suggestion
{: .bs-callout .bs-callout-info}The footprint analyzer is currently released as a beta feature, meaning that some of the information may not be completely accurate or present at all. Beta status of the feature is also indicated on the product listing page.
```

> {: .bs-callout .bs-callout-info} The footprint analyzer is currently released as a beta feature, meaning that some of the information may not be completely accurate or present at all. Beta status of the feature is also indicated on the product listing page.

Reviewer 2: ```suggestion
{:.bs-callout-info}
The footprint analyzer is a beta feature, meaning that some of the information may not be completely accurate or present at all. The beta status of the feature is also indicated on the product listing page.
```

## Overview

> Foorptint analysis is an automated check that generates report based on static analysis of the extension code. After processing, reports are visible on the product listing page and include information on certain specific features which provide Magento Marketplace customers insights on extension capabilities from the code perspective.

Reviewer: ```suggestion
Footprint analysis is an automated check that generates a report based on the static analysis of the extension code. After processing, reports are visible on the product listing page and include information on certain specific features which provide Magento Marketplace customers insights on extension capabilities from the code perspective.
```

> Footprint analysis is an automated check that generates a report based on the static analysis of the extension code. After processing, reports are visible on the product listing page and include information on certain specific features which provide Magento Marketplace customers insights on extension capabilities from the code perspective.

Reviewer 2: ```suggestion
Footprint analysis is an automated check that generates a report based on the static analysis of the extension code. After processing, reports are visible on the product listing page and include information on certain specific features, which provide Magento Marketplace customers insights on extension capabilities from the code perspective.
```

## When testing is done

> All extension packages have their footprint analyzed at the time of submissions. This check is not a test and therefore cannot be passed or failed.

Reviewer: ```suggestion
All extension packages have their footprint analyzed at the time of submission. This check is not a test and therefore cannot be passed or failed.
```

## What is being checked

> At the moment Footptint Analyzer check is performing static package code testing to identify APIs exposed by the module. Report usually consists of the following sections:

Reviewer: ```suggestion
At the moment the footprint analyzer check is performed on the static package code to identify APIs exposed by the module. The report usually consists of the following sections:
```

> At the moment the footprint analyzer check is performed on the static package code to identify APIs exposed by the module. The report usually consists of the following sections:

Reviewer 2: ```suggestion
At the moment, the footprint analyzer check is performed on the static package code to identify APIs exposed by the module. The report usually consists of the following sections:
```

> *  Programmatic API and Data interfaces – provides information on the number of programmatic APIs found in the extension code package.

Reviewer 2: ```suggestion
*  Programmatic API and data interfaces—Provides information on the number of programmatic APIs found in the extension code package.
```

> *  Web API – provides insights on the number of REST and SOAP APIs exposed by the extension package.

Reviewer 2: ```suggestion
*  Web API—Provides insights on the number of REST and SOAP APIs exposed by the extension package.
```

> *  GraphQL – verifies if extension exposes any GraphQL entities.

Reviewer: ```suggestion
*  GraphQL – verifies if the extension exposes any GraphQL entities.
```

> *  GraphQL – verifies if the extension exposes any GraphQL entities.

Reviewer 2: ```suggestion
*  GraphQL—Verifies if the extension exposes any GraphQL entities.
```
